src/main/cpp/VideoState.cpp

```diff
@@ -1121,24 +1109,14 @@ int VideoState::stream_start() {
 
 	realtime = is_realtime(ic);
 
-	if (show_status)
+	if (ENABLE_SHOW_STATUS) {
```

Comment:
Same here (see VideoState.h comment on line 75)

Reply:
Changed to use const